docs/docs-content/clusters/cluster-management/cluster-updates.md

```diff
@@ -140,3 +142,123 @@ You can follow these steps to validate all cluster update approaches.
 3. Select the cluster you updated, and navigate to the **Profile** tab of the cluster.
 
 4. Confirm that the cluster is now using an updated profile.
+
+## Palette Reconciliation Behavior
```

Comment:
Is this heading appropriate? It sounds like you are providing a how-to on "Remove redundant cluster resources after update" instead of explaining the behavior. I would maybe use that as the heading.